### PR TITLE
Co 2015 sms child reservation failure

### PR DESCRIPTION
**Pull request by nbornand**

### CODE REVIEW

**Comment by ecino** [comment]
Can you tell me if/when this is ready to be reviewed?

**Comment by nbornand** [comment]
Should be fine, I'm not sure why you added the "need fixes" yesterday

**Comment by ecino** [comment]
Because of conflicting files. You could resolve them by rebasing your branch.

File: sbc_compassion/models/correspondence.py [hunk 1]
```diff
@@ -276,17 +276,17 @@ def _set_partner_review(self):
     @api.depends('page_ids')
     def _compute_original_text(self):
         for letter in self:
-            letter.original_text = letter._get_text('original_text')
+            letter.original_text = letter._get_sms_message('original_text')
```
Comment (ecino) [inline sbc_compassion/models/correspondence.py:279]: This is a bad search and replace in my opinion

File: sbc_compassion/models/correspondence.py [hunk 2]
```diff
@@ -276,17 +276,17 @@ def _set_partner_review(self):
...
 
     @api.depends('page_ids')
     def _compute_translated_text(self):
         for letter in self:
-            letter.translated_text = letter._get_text('translated_text')
+            letter.translated_text = letter._get_sms_message('translated_text')
```
Comment (ecino) [inline sbc_compassion/models/correspondence.py:284]: This is a bad search and replace in my opinion

File: sbc_compassion/models/correspondence.py [hunk 3]
```diff
@@ -276,17 +276,17 @@ def _set_partner_review(self):
...
 
     @api.depends('page_ids')
     def _compute_english_translated_text(self):
         for letter in self:
-            letter.english_text = letter._get_text('english_translated_text')
+            letter.english_text = letter._get_sms_message('english_translated_text')
```
Comment (ecino) [inline sbc_compassion/models/correspondence.py:289]: This is a bad search and replace in my opinion

**Comment by ecino** [comment]
I think we will put this on hold as the process is likely to change based on what we discussed with David today, having first a set of children already available and only on high demand fetching new children by batches.

**Pull request closed without merging**